src/components/App/ExpressRecorder.tsx

```diff
@@ -396,7 +396,7 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
     recordAgain = () => {
         this.setState({
             recordedBlobs: [],
-            doCountdown: true,
+            doCountdown: false,
```

Comment:
I think doCountdown will already be false at that time. you can just remove this line completely, IMO.

Reply:
yes, seems to be working